geopandas/sindex.py

```diff
@@ -568,6 +568,17 @@ def query_bulk(self, geometry, predicate=None, sort=False):
 
             return res
 
+        def nearest_all(self, geometry, max_distance=None, return_distance=False):
```

Comment:
It looks like you need to stub this docstring out in the base class, then add
```
@doc(BaseSpatialIndex.nearest_all)
```

Reply:
Do we even want `nearest_all` to be part of the base spatial index? I'm not sure...